R/modify_pd.R

```diff
@@ -0,0 +1,43 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler] or
+#'   [create_filler_nested].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  start <- which(pd$token == "'('") + 1
+  stop <- which(pd$token == "')'") - 1
+  if (length(start) == 0 && length(stop) == 0) {
```

Comment:
I am not sure I understand what you mean. Probably it's enough to check for just one of them to be zero.